pulsar-common/src/main/java/org/apache/pulsar/common/functions/FunctionState.java

```diff
@@ -34,6 +34,7 @@
 public class FunctionState {
     private String key;
     private String stringValue;
+    private byte[] byteValue;
```

Comment:
Is this relevant to this PR?

Reply:
For both get and put state, we need to know if we are dealing with byte values or string. The current version only assumes string. Hence the change